assets/app/scripts/controllers/deployments.js

```diff
@@ -94,6 +345,20 @@ angular.module('openshiftConsole')
       });
     });
 
+    function getErrorDetails(result) {
```

Comment:
this seems like it should be a common function and not specific to this controller